src/instructlab/data/generate.py

```diff
@@ -34,11 +34,19 @@
     default=config.DEFAULT_CHUNK_WORD_COUNT,
     show_default=True,
 )
+# TODO - DEPRECATED - Remove in a future release
 @click.option(
     "--num-instructions",
     type=click.INT,
-    help="Number of instructions to generate.",
-    default=config.DEFAULT_NUM_INSTRUCTIONS,
+    help="DEPRECATED. See --sdg-scale-factor",
```

Comment:
If the flag is hidden will it be shown in the help? Also, the deprecated message is not needed if we use `deprecated=True`.

Reply:
I don't think it's shown in the help output, so maybe this is never used anywhere. I'll check

Reply:
dropped the help text